app/src/main/res/layout/fragment_feedback_recyclerview_row_layout.xml

```diff
@@ -0,0 +1,128 @@
+                app:layout_constraintTop_toTopOf="@+id/feedback_row_layout_likeCounts_textView"
+                app:layout_constraintVertical_bias="0.0" />
+
+            <ImageView
+                android:id="@+id/feedback_row_layout_comments_imageview"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_50sdp"
+                android:scaleType="centerCrop"
+                app:layout_constraintBottom_toBottomOf="@+id/feedback_row_layout_likes_textview"
+                app:layout_constraintStart_toEndOf="@+id/feedback_row_layout_likes_textview"
+                app:layout_constraintTop_toTopOf="@+id/feedback_row_layout_likes_textview"
+                app:layout_constraintVertical_bias="0.6"
+                app:srcCompat="@drawable/ic_sms"
+                app:tint="@color/black"
+                 />
+
+            <TextView
+                android:id="@+id/feedback_row_layout_comments_textview"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_5sdp"
+                android:text="@string/comment"
+                app:layout_constraintBottom_toBottomOf="@+id/feedback_row_layout_commentsCounts"
+                app:layout_constraintStart_toEndOf="@+id/feedback_row_layout_commentsCounts"
+                app:layout_constraintTop_toTopOf="@+id/feedback_row_layout_commentsCounts" />
+
+            <TextView
+                android:id="@+id/feedback_row_layout_likeCounts_textView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_5sdp"
+                android:text="0"
+                android:textColor="@color/red"
+                app:layout_constraintBottom_toBottomOf="@+id/feedback_row_layout_likes_imageview"
+                app:layout_constraintStart_toEndOf="@+id/feedback_row_layout_likes_imageview"
+                app:layout_constraintTop_toTopOf="@+id/feedback_row_layout_likes_imageview" />
+
+
+            <TextView
+                android:id="@+id/feedback_row_layout_commentsCounts"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_5sdp"
+                android:text="@string/_0"
+                app:layout_constraintBottom_toBottomOf="@+id/feedback_row_layout_comments_imageview"
+                app:layout_constraintStart_toEndOf="@+id/feedback_row_layout_comments_imageview"
+                app:layout_constraintTop_toTopOf="@+id/feedback_row_layout_comments_imageview"
+                app:layout_constraintVertical_bias="0.4" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+    </androidx.cardview.widget.CardView>
+
```

Comment:
You can have a text view and drawable left instead of textview and image view separately